src/redisraft.h

```diff
@@ -308,92 +307,92 @@ void fsyncThreadAddTask(FsyncThread *th, int fd, raft_index_t requested_index);
-
-    raft_node_id_t id;                  /* Local node Id */
-    NodeAddr addr;                      /* Address of local node, if specified */
-    char *rdb_filename;                 /* Original Redis dbfilename */
-    char *log_filename;                 /* Raft log file name, derived from dbfilename */
-    bool follower_proxy;                /* Do follower nodes proxy requests to leader? */
-    bool quorum_reads;                  /* Reads have to go through quorum */
-    char *ignored_commands;             /* Comma delimited list of commands that should not be intercepted */
-    char *cluster_user;                 /* ACL user to use for internode communication */
-    char *cluster_password;             /* Password used for internode communication */
+    RedisModuleString *str_conf_ref; /* Reference of the last string config that we pass to Redis module API */
+
+    raft_node_id_t id;      /* Local node Id */
+    NodeAddr addr;          /* Address of local node, if specified */
+    char *rdb_filename;     /* Original Redis dbfilename */
+    char *log_filename;     /* Raft log file name, derived from dbfilename */
+    bool follower_proxy;    /* Do follower nodes proxy requests to leader? */
+    bool quorum_reads;      /* Reads have to go through quorum */
+    char *ignored_commands; /* Comma delimited list of commands that should not be intercepted */
+    char *cluster_user;     /* ACL user to use for internode communication */
+    char *cluster_password; /* Password used for internode communication */
 
     /* Tuning */
-    int periodic_interval;              /* raft_periodic() interval */
-    int request_timeout;                /* Milliseconds before sending a heartbeat message to the followers */
-    int election_timeout;               /* Milliseconds before starting an election if there is no leader */
-    int connection_timeout;             /* Milliseconds the node will continue to try connecting to another node */
-    int join_timeout;                   /* Milliseconds the node will continue to try joining a cluster */
-    int reconnect_interval;             /* Milliseconds to wait to reconnect to a node if connection drops */
-    int proxy_response_timeout;         /* Milliseconds to wait for a response to a proxy request */
-    int response_timeout;               /* Milliseconds to wait for a response to a Raft message */
-    long long append_req_max_count;     /* Max in-flight appendreq message count between two nodes. */
-    long long append_req_max_size;      /* Max appendreq message size in bytes. Just an approximation. */
-    long long snapshot_req_max_count;   /* Max in-flight snapshotreq message count between two nodes. */
-    long long snapshot_req_max_size;    /* Max snapshotreq message size in bytes. Just an approximation. */
-    long long scan_size;                /* how many keys to fetch at a time internally for raft.scan */
+    int periodic_interval;            /* raft_periodic() interval */
```

Comment:
Consecutive comment lines are aligned but we cannot align with the upper section anymore. (There is a new line between them).